content/shmem_malloc.tex

```diff
@@ -28,14 +28,19 @@
     The \FUNC{shmem\_malloc} routine returns a pointer to a block of at least
     \VAR{size} bytes suitably aligned for any use.  This space is allocated from the
     symmetric heap (in contrast to \FUNC{malloc}, which allocates from the private
-    heap).
+    heap).  When \VAR{size} is zero, the \FUNC{shmem\_malloc} routine performs
+    no action and returns a null pointer.
     
     The \FUNC{shmem\_align} routine allocates a block in the symmetric heap that has
-    a byte alignment specified by the \VAR{alignment} argument.
+    a byte alignment specified by the \VAR{alignment} argument.  The value of
+    \VAR{alignment} shall be a multiple of \CONST{sizeof(void *)}, that is also
```

Comment:
Can you remove the comma? Then it will read:

> The value of `alignment` shall be a multiple of `sizeof(void *)` that is also a power of two.

With the comma, it seems _possible_ that one could read the sentence as if it said:

> The value of `alignment` shall be a multiple of `sizeof(void *)`; that is, also a power of two.

which would not be correct.

Reply:
Done -- ec57fedb